packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -142,6 +142,11 @@ export interface DataGridPropsWithDefaultValues<R extends GridValidRowModel = an
    * @default true
    */
   rowSelection: boolean;
+  /**
+   * The milliseconds throttle delay for resizing the grid.
+   * @default 60
+   */
+  resizeThrottleMs: number;
```

Comment:
Nice addition, I think adding a line or two about it [here](https://mui.com/x/react-data-grid/column-dimensions/#resizing) may help some users even further.